habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -233,6 +242,9 @@ class ArmRelPosKinematicReducedActionStretch(RobotAction):
     def __init__(self, *args, config, sim: RearrangeSim, **kwargs):
         super().__init__(*args, config=config, sim=sim, **kwargs)
         self.last_arm_action = None
+        self._delta_pos_limit = self._config.delta_pos_limit
```

Comment:
A lot of code refactoring could be safe here by just converting the omega config to the dataclass here and access those members, no?

Reply:
I will leave this refactor in place, just because I think it makes more sense. 
I instantiated the configs for the RobotActions, but leaving the measures and sensors as is because they all use the config differently.